icon/iiss/prepmanager.go

```diff
@@ -298,34 +308,25 @@ func (pm *PRepManager) ChangeDelegation(od, nd icstate.Delegations) error {
 
 	delegatedToNotReadyNode := big.NewInt(0)
 	var newPs *icstate.PRepStatus
-	for k, v := range delta {
-		owner, err := common.NewAddress([]byte(k))
+	for key, value := range delta {
+		owner, err := common.NewAddress([]byte(key))
 		if err != nil {
 			return err
 		}
 
-		key := icutils.ToKey(owner)
-		if delta[key].Cmp(icstate.BigIntZero) != 0 {
+		if value.Cmp(icstate.BigIntZero) != 0 {
 			ps := pm.state.GetPRepStatus(owner)
 			if ps == nil {
 				// Someone tries to set delegation to a PRep which has not been registered
 				newPs = icstate.NewPRepStatus(owner)
 				newPs.SetStatus(icstate.NotReady)
-				delegatedToNotReadyNode.Add(delegatedToNotReadyNode, delta[key])
+				delegatedToNotReadyNode.Add(delegatedToNotReadyNode, value)
 			} else {
 				newPs = ps.Clone()
 			}
 
-			newPs.Delegated().Add(newPs.Delegated(), v)
-
-			if newPs.Status() == icstate.NotReady && newPs.Delegated().Cmp(icstate.BigIntZero) == 0 {
```

Comment:
```go
newPs.Status() == icstate.NotReady
```
This code makes panic when ICONist clear a delegation for not registered P-Rep

```go
newPs.Delegated().Cmp(icstate.BigIntZero) == 0
```
This does not mean PRepStatus is empty, so I delete this if statement code block